src/chariott_grpc.rs

```diff
@@ -4,6 +4,14 @@
 use std::sync::{Arc, RwLock};
 use std::time::Instant;
 
+use chariott_proto::{
+    common::intent::Intent,
+    runtime::{
+        chariott_service_server::ChariottService, AnnounceRequest, AnnounceResponse,
+        FulfillRequest, FulfillResponse, IntentRegistration, IntentServiceRegistration,
+        RegisterRequest, RegisterResponse, RegistrationState,
+    },
```

Comment:
nit: Since it imports so many types directly linked to a module to do with gRPC, perhaps worth simplifying to:

```suggestion
    runtime::{chariott_service_server::ChariottService, *},
```

Reply:
Thanks for the suggestion, I would [prefer to not use glob imports](https://github.com/eclipse/chariott/pull/32#discussion_r1022596092)